test/test_transforms_tensor.py

```diff
@@ -718,5 +626,122 @@ def setUp(self):
         self.device = "cuda"
 
 
+def _test(device, **kwargs):
+    tensor = torch.randint(0, 256, size=(3, 44, 56), dtype=torch.uint8, device=device)
+    batch_tensors = torch.randint(0, 256, size=(4, 3, 44, 56), dtype=torch.uint8, device=device)
+    transform = T.RandomAffine(**kwargs)
+    s_transform = torch.jit.script(transform)
+
+    _test_transform_vs_scripted(transform, s_transform, tensor)
+    _test_transform_vs_scripted_on_batch(transform, s_transform, batch_tensors)
+
+    return s_transform
+
+
+@pytest.mark.parametrize('device', cpu_and_gpu())
+def test_random_affine(device):
+    s_transform = _test(device, degrees=0.0)
```

Comment:
I realize this was already present, but we don't need to get `s_transform` from `_test()`. `_test()` doesn't need to return anything and we can just declare `s_transform` here as `s_transform = torch.jit.scriptT.RandomAffine(...)`

(where `...` may actually be empty)